client/scripts/helpers/chain_notification_types.ts

```diff
@@ -175,3 +175,35 @@ export const KulupuChainNotificationTypes = {
     'kulupu-vote-delegated',
   ],
 };
+
+export const DydxChainNotificationTypes = {
+  Governance: [
+    'proposal-canceled',
+    'proposal-created',
+    'proposal-executed',
+    'proposal-queued',
+    'vote-emitted',
+  ],
+  Token: [
```

Comment:
These token ones aren't going to work given the way the contracts are designed.

Reply:
Can you say more?